### ISSUE TITLE
upgrade to mongo@3

### ISSUE THREAD

**Issue by mcollina** [fastify/fastify-mongodb#24]
mongo@3 has been a breaking release.

Can someone figure it out and update this?

**Comment by allevo** [comment]
The changes are summarized [here](https://github.com/mongodb/node-mongodb-native/blob/3.0.0/CHANGES_3.0.0.md).

I'm 👍  particularly because the new version 3.0 support the mongo db server 3.6 too (released 1 month ago more or less).

For the CRUD operations, nothing changes.

The main breaking is during the connection.

From the link above:
```js
MongoClient.connect('mongodb://localhost:27017/test', (err, db) => {
  // Database returned
});
```
become
```js
MongoClient.connect('mongodb://localhost:27017/test', (err, client) => {
  // Client returned
  var db = client.db('test');
});
```

**Comment by allevo** [comment]
#25

**Comment by fox1t** [comment]
Isn't this feature landed?

**Comment by allevo** [comment]
Hi @fox1t !
This feature is merged https://github.com/fastify/fastify-mongodb/pull/25 and is landed with [0.5.0](https://github.com/fastify/fastify-mongodb/releases/tag/v0.5.0)
